modules/ROOT/pages/query-unified-schema.adoc

```diff
@@ -21,7 +25,70 @@ Choose one of the following options:
 ** *I already have credentials...*: Click *Next*, provide an existing client ID and secret pair, and then click *Authenticate & Request Access*.
 . To change your access method from the *Run Query* page, click *Manage Credentials* > *Switch credentials*.
 
-=== Request Access to Environments Using an External Client Provider
+
+== Write and Run a Query
+
+Use the query editor to write, run, and troubleshoot queries. 
+
+. If any of the APIs in the unified schema require pass-through authentication, click *Add HTTP Headers*, and:
```

Comment:
@mauro-dentoni this is for adding pass-through auth before running a query

Reply:
The statement looks fine Isaac. We will enforce the add of the headers, before running a query, if any of the APIs requires pass-through authentication. Thanks